src/analysis/convex/cone.lean

```diff
@@ -394,6 +394,25 @@ lemma salient_strictly_positive : salient (strictly_positive 𝕜 E) :=
 lemma blunt_strictly_positive : blunt (strictly_positive 𝕜 E) := lt_irrefl 0
 
 end positive_cone
+
+section ordered_semiring
```

Comment:
Can this go in the earlier ordered_semiring section?

Reply:
Should I move the positive cone into the positive semiring section as well?

Reply:
Ah nvm, I see that there are several additional conditions on `k`.